### PR TITLE
Update the Getting Started Chinese document to avoid the novice step on the pit

### PR DESCRIPTION
**Pull request by yl-yue**
Update the Getting Started Chinese document to avoid the novice step on the pit

### CODE REVIEW

File: docs/zh-CN/server/exception-handling.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+本节介绍如何处理 `GrpcService` 层内的异常，类似 `spring-boot-web` 中的统一异常处理。
+
+## 目录 <!-- omit in toc -->
+
+- [正确的异常处理](#proper-exception-handling)
+- [详细解释](#detailed-explanation)
+  - [映射异常的优先级](#priority-of-mapped-exceptions)
+  - [发送元数据作为响应](#sending-metadata-in-response)
+  - [可返回类型概述](#overview-of-returnable-types)
```
Comment (ST-DDT) [inline docs/zh-CN/server/exception-handling.md:13]: I think you have to update the fragment identifier to use chinese titles.

Comment (yl-yue) [reply docs/zh-CN/server/exception-handling.md:13]: Thank you for reminding, the title index has been updated

**Comment by yidongnan** [comment]
Thank you for your translation. It would be better if you are willing to manage the translation of Chinese documents through crowdin. The crowdin tool can effectively improve the efficiency of translation and proofreading. I will send you an invitation link via email later.

**Comment by yidongnan** [comment]
I have translated this part in crowdin

**Pull request closed without merging**